src/components/streams/TrendingStreamsContainer.js

```diff
@@ -6,7 +6,7 @@ import TrendingStreams from './TrendingStreams'
 import { loadTrendingStreams } from '../../actions/stream'
 
 function mapStateToProps (state, ownProps) {
-  const streams = at(state.stream, state.stream.trending)
+  const streams = at(state.stream, state.stream.trending || [])
```

Comment:
Do we need to worry about trending streams when it is empty? Should we worry about state.stream?